src/main/java/duke/command/CommandType.java

```diff
@@ -0,0 +1,66 @@
+package duke.command;
+import java.util.HashMap;
+import java.util.Map;
+
+import duke.DukeException;
+
+/**
+ * An Enum class to represent the types of commands a user can perform.
+ */
+public enum CommandType {
+    BYE("bye", null),
+    LIST("list", null),
+    MARK("mark", "\\d+"),
+    UNMARK("unmark", "\\d+"),
+    DELETE("delete", "\\d+"),
+    FIND("find", null),
+    TODO("todo", null),
+    EVENT("event", "/at"),
+    DEADLINE("deadline", "/by");
+
+    private static final Map<String, CommandType> commandMap = new HashMap<>();
```

Comment:
```suggestion
    private static final Map<String, CommandType> COMMAND_MAP = new HashMap<>();
```
Please capitalise your constant variables. I noticed that there are similar issues in other parts of your code too